src/index.js

```diff
@@ -10,8 +10,19 @@ async function exec () {
   try {
     const prefixes = core.getInput('prefixes')
     const refBranch = core.getInput('refBranch')
-    const token = core.getInput('GITHUB_TOKEN')
-    const octokit = github.getOctokit(token)
+    const github_token = core.getInput('GITHUB_TOKEN')
+    const merge_token = core.getInput('MERGE_TOKEN')
+
+    const octokit = github.getOctokit(github_token)
+
+    let merge_octokit
+    if (merge_token) {
```

Comment:
These new variables need to use camelCase style. That's why liniting is failing

Reply:
RIP Python I miss you. Can't wait to go back to it!